README.md

```diff
@@ -119,4 +119,4 @@ make itest
 Individual itests can also be run using:
 ```
 ./run_itest.sh {test name}
-```
\ No newline at end of file
+```
```

Comment:
nit: unrelated change on this line?

Reply:
Hm this was something GitHub UI added? Let me try and clean it up

Reply:
Could be a ghost-diff, I've seen them before 👻 no need to sink too much time into it!

Reply:
👍